src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -1962,16 +1992,6 @@ emit_vector_2_3_4 (MonoCompile *cfg, MonoMethod *cmethod, MonoMethodSignature *f
 		ins->inst_c1 = ty;
 		return ins;
 	}
-	case SN_GetElement: {
```

Comment:
This was moved to be an extension method hanging off `static class Vector` (which is likewise where it would live if we eventually make it public), so I moved the logic up to handle it.

Reply:
Same for `SN_WithElement` below

Reply:
I can see `SN_WithElement` case is not being removed. Based on what you said earlier, they should be gone too?

Reply:
Did this locally, forgot to `git add`. Will be included in the commit pushed after extra-platforms finishes.